ontology/yaml/resources/HVAC/entity_types/ADY.yaml

```diff
@@ -29,3 +29,16 @@ ADY_BS_SS:
   - control_mode
   - schedule_run_command
   - control_status
+
+AION_BS_SS:
+  description: "Basic air ionizer with trip alarm, switch control/status, scheduled run, filter alarm."
+  is_canonical: true
+  implements:
+  - AION
+  - SS
+  opt_uses:
+  - lost_power_alarm
+  - control_mode
+  - schedule_run_command
```

Comment:
If you intend this to be an abstract type (you have defined it in this PR) then you should require it. If you do have an ionizer that doesnt have a schedule, you should have two separate canonical types -- one with and one without that abstract type.

Reply:
thanks sir. I like it to be abstract and define 2 canonical ones for maximum flexibility.
Is it ok to make the 2 canonical types from abstract (one with schedule and one without) even if I only know and use the one with schedule? 
As of now, I don't know if there is ionizer without schedule. Do I just make 1 canonical device (with schedule) that we use and let others define the other later (without schedule) for their own use?

Reply:
it is okay to suggest both if you think it likely you will run into it in the field. something like this:
```
AION_SS:
  implements:
  - AION
  - SS
  opt_uses:
  - lost_power_alarm
  - control_mode

AION_SS_SRC:
  implements:
  - AION
  - SS
  - SRC
  opt_uses:
  - lost_power_alarm
  - control_mode
```

if you wanted to omit the one without schedule we can also do that. it might be easiest just to have both. i am okay with either one.

Reply:
one more thing: please note the names for each type above is constructed from the implemented abstract types. you will find some that do not follow this convention, but in general all canonical types should try to follow that convention. just FYI in case you make more canonical types.

Reply:
noted sir.  I changed some more items to reflect this. thanks.